### PR TITLE
[Feat]: Implement auto antijam

### PR DESCRIPTION
**Pull request by ngao4**
1. add auto antijam
2. change the parameter to avoid the overheat problem
3. add the dbus channel for new shooting mode and add the instruction in config.md

### CODE REVIEW

**Comment by ngao4** [comment]
LGTM!

File: shared/libraries/shooter.h [hunk 1]
```diff
@@ -22,6 +22,7 @@
 
 #include "controller.h"
 #include "motor.h"
+#include "dbus.h"
```
Comment (AzulRadio) [inline shared/libraries/shooter.h:25]:
```suggestion

```

Is this header necessary?

File: shared/libraries/dbus.h [hunk 2]
```diff
@@ -24,6 +24,10 @@
 
 namespace remote {
 
+const uint16_t dbus_wheel_upperbound = 6960;
+const uint16_t dbus_wheel_digital_lowerbound = 1684;
+const uint16_t dbus_wheel_mid = 1024;
```
Comment (AzulRadio) [inline shared/libraries/dbus.h:29]: Please use uppercase for constants.

**Review by AzulRadio — APPROVED** [review]
LGTM

**Pull request merged**